pkgs/default.nix

```diff
@@ -56,3 +56,9 @@ let
   );
 in
 finalPkgs
+// {
+  cosmic-ext-examine = pkgs.lib.warn "cosmic-ext-examine has been renamed to examine." finalPkgs.examine;
```

Comment:
These should be conditional on `config.allowAliases`, but ideally it can be separated out into an aliases.nix file. I'm happy to do this myself and it can be considered out-of-scope of this PR if you'd rather that

Reply:
Moved it to `aliases.nix`